Deployment-Instructions.md

```diff
@@ -70,23 +66,33 @@ The [stage.sh](stage.sh) script at the root of the repository must be used to up
 Example on how to stage your project to a custom bucket and path (note the path is optional but, if specified, must end with '/'):
 
 ```bash
-./stage.sh MY_CUSTOM_BUCKET S3_PATH/ --private-s3 --skip-virtualenv
+./stage.sh MY_CUSTOM_BUCKET S3_PATH/ --skip-virtualenv
 ```
-NB:   --private-s3 is required to deploy to a private bucket (this is the recommended approach)
 
-The stage script will output a path to your master deployment CloudFormation template.  You can use this link to your S3 bucket to start a new deployment via the CloudFormation console in your AWS Console or use the command line below.  (replace REGION, MY_CUSTOM_BUCKET and S3_PATH value)
+### Step 5: Deploy the Cloudformation template
```

Comment:
Should be "##" (h2) rather than "###" (h3) to match header size of prior steps.

Nit: "CloudFormation"